L/Libxc/Libxc_GPU/build_tarballs.jl

```diff
@@ -5,42 +5,53 @@ include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
 include(joinpath(YGGDRASIL_DIR, "platforms", "cuda.jl"))
 
 name = "Libxc_GPU"
-version = v"6.1.0"
+version = v"7.0.0"
 include("../sources.jl")
 
 sources = [
     sources;
-    DirectorySource("./bundled")
+    # aarch64 binaries precompiled and hosted on GitHub
```

Comment:
Perhaps add a comment here explaining why this is done instead of building them in the recipe?